<?xml version="1.0"?>
<launch>

    <arg name="start_pt" default= '0.3888, -3.173, 1.5790375, 0'/>
    <arg name="use_rviz" default="true"/>
    <arg name="is_sim" default="true"/>
    <arg name="map_file" default="$(find svea_core)/resources/maps/itrl.yaml" />

    <group if="$(eval use_rviz == true)">
        <node type="rviz" name="rviz" pkg="rviz" args="-d $(find svea_core)/resources/rviz/SVEA_itrl.rviz" />
    </group>

    <group if="$(eval is_sim == true)">
        <node pkg="map_server" type="map_server" name="map_server" output="screen" args="$(arg map_file)" />
    </group>

    <node pkg="svea_core" type="itrl_example.py" name="itrl_example" output="screen">
        <param name="start_pt" value="$(arg start_pt)"/>
        <param name="use_rviz" value="$(arg use_rviz)"/>
        <param name="is_sim" value="$(arg is_sim)"/>
    </node>

</launch>
